app/api/items/service.py

```diff
@@ -48,3 +48,14 @@ def update_item(db: Session, item_id: UUID4, payload: ItemUpdate):
 
     updated_item = get_item_by_item_id(db, item_id)
     return updated_item
+
+
+def add_photos(db: Session, item_id: UUID4, photos: conlist(UUID4, min_items=1)):
```

Comment:
```suggestion
def add_photos(db: Session, item_id: UUID4, new_photo_ids: conlist(UUID4, min_items=1)):
```

不要動到函式的參數（後面的程式碼有將 photos 重新賦值）